tests/setup.py

```diff
@@ -38,7 +38,7 @@ def finalize_options(self):
     def run_tests(self):
         # import here, cause outside the eggs aren't loaded
         import pytest
-        print self.pytest_args
+        print(self.pytest_args)
```

Comment:
This was not valid python3. Maybe everyone else ran setup.py in python2 and haven't run it again since?

Reply:
note that it's also valid python2 to use parens here, so I'd advocate for keeping this change.